packages/cli/src/create/command.ts

```diff
@@ -15,6 +15,7 @@ let projectPath: string = "";
 let framework: string = "";
 let language: string = "";
 let baseContract: string = "";
+let chain: string = "";
```

Comment:
if this is either `solana` or empty then why not make it a boolean value? `isSolana`